sdk/dotnet/GetCassanda.cs

```diff
@@ -6,6 +6,7 @@
 using System.Collections.Immutable;
```

Comment:
This file name is misspelled - should be Cassandra.

Reply:
We will follow behind on that :)